src/axolotl/utils/collators.py

```diff
@@ -0,0 +1,121 @@
+"""
+DataCollator for axolotl to pad labels and position_ids for packed sequences
+"""
+from dataclasses import dataclass
+from typing import Any, Optional, Union
+
+import numpy as np
+from transformers import PreTrainedTokenizerBase
+from transformers.utils import PaddingStrategy
+
+
+@dataclass
+class DataCollatorForSeq2Seq:
```

Comment:
Should this inherit from Transformers?

Reply:
It was a full copy/paste and I don't think there is anything I wanted to inherit